test_scripts/API/Restructuring_OnResetTimeout/002_BC_OnResetTimeout_Success_flow_without_resetPeriod.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(without resetPeriod) to SDL
```

Comment:
I suggest to update the text to 
3) HMI sends BC.OnResetTimeout(without resetPeriod) to SDL at 5 seconds after HMI received the request 

because the code says `notificationTime = 5000`